### PR TITLE
Add flutter create for the web

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Allows creating the web specific bits - currently just an index.html, behind a flag `flutter create --web`

test: manual

### CODE REVIEW

File: packages/flutter_tools/templates/app/web/index.html.tmpl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<head>
```
Comment (christopherfujino) [inline packages/flutter_tools/templates/app/web/index.html.tmpl]: Missing an opening `<html>` tag here?

Comment (jonahwilliams) [reply packages/flutter_tools/templates/app/web/index.html.tmpl]: yes...but it worked...yay html!

Comment (christopherfujino) [reply packages/flutter_tools/templates/app/web/index.html.tmpl]: LOL, yeah...

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**